api/src/main/java/io/strimzi/api/kafka/model/bridge/KafkaBridgeConsumerSpec.java

```diff
@@ -21,16 +21,39 @@
         builderPackage = Constants.FABRIC8_KUBERNETES_API
 )
 @JsonInclude(JsonInclude.Include.NON_NULL)
-@JsonPropertyOrder({"config"})
+@JsonPropertyOrder({"enabled", "timeoutSeconds", "config"})
 @EqualsAndHashCode(callSuper = true)
 @ToString(callSuper = true)
 public class KafkaBridgeConsumerSpec extends KafkaBridgeClientSpec {
     public static final String FORBIDDEN_PREFIXES = "ssl., bootstrap.servers, group.id, sasl., security.";
     public static final String FORBIDDEN_PREFIX_EXCEPTIONS = "ssl.endpoint.identification.algorithm, ssl.cipher.suites, ssl.protocol, ssl.enabled.protocols";
 
+    public static final long HTTP_DEFAULT_TIMEOUT = -1L;
+
+    private boolean enabled = true;
+    private long timeoutSeconds = HTTP_DEFAULT_TIMEOUT;
+
     @Override
     @Description("The Kafka consumer configuration used for consumer instances created by the bridge. Properties with the following prefixes cannot be set: " + FORBIDDEN_PREFIXES + " (with the exception of: " + FORBIDDEN_PREFIX_EXCEPTIONS + ").")
     public Map<String, Object> getConfig() {
         return config;
     }
+
+    @Description("Whether the HTTP consumer should be enabled or disabled")
+    public boolean isEnabled() {
+        return enabled;
+    }
+
+    public void setEnabled(boolean enabled) {
+        this.enabled = enabled;
+    }
+
+    @Description("The timeout in seconds for deleting inactive consumers.")
+    public long getTimeoutSeconds() {
```

Comment:
Same as above, if you want to use `long` with default value, you should add `@JsonInclude(JsonInclude.Include.NON_DEFAULT)`.